### ISSUE TITLE
-BUG if not help me 

### ISSUE THREAD

**Issue by Mothanick** [finndev/PokeBuddy#440]
i have used this program before and everything worked fine. and wen it was updated 12 hours ago i downloaded it again it it was great but an hour later its saying i have collected 0 exp from pokestops please help me

**Comment by 311guy** [comment]
Let me know if you figure this out plz

**Comment by WilliamcaFox** [comment]
Typically after moving to a new location you will get a soft ban for about 5 minutes, especially if you recently signed in at another location. It's Niantics soft ban method.

**Comment by Ander0128** [comment]
What exactly is a soft ban? Just a automatic temp ban?

**Comment by WilliamcaFox** [comment]
Yes, a soft ban is just an automatic temporary ban.

**Comment by Mothanick** [comment]
thank you i was very confused :)

**Comment by 311guy** [comment]
How long does this last?

**Comment by WilliamcaFox** [comment]
Typically about 5 minutes it seems.

**Comment by 311guy** [comment]
Mines been like this over an hour now.

**Comment by WilliamcaFox** [comment]
Tried a stop and restart / fresh install? After running mine for a while last night I had to restart 'em in the morning.

**Comment by 311guy** [comment]
Yup deleted all files, reinstalled, running two windows and by other acct is fine

**Comment by WilliamcaFox** [comment]
Try signing into your account through your phone with gps and network scanning always enabled off or through Nox.
